src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -179,11 +180,13 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ItemGroup>
 
     <!--
-        Not using SkipUnchangedFiles="true" because the application may want to change
-        one of these files and not have an incremental build replace it.
-        -->
+      Use SkipUnchangedFiles to prevent unnecessary file copies. The copy will occur if the
+      destination doesn't exist, the source is newer than the destination, or if the source and
```

Comment:
I *think* it is technically newer or older, just different timestamp or different size. cc @rainersigwald 

But this is fine.

Reply:
https://github.com/microsoft/msbuild/blob/95862314f14733dd0086b1747264c40e6989f2cf/src/Tasks/Copy.cs#L136-L164

Reply:
👍